knotx-core/knotx-service-adapter/src/main/java/com/cognifide/knotx/core/serviceadapter/ServiceAdapterResponse.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Knot.x - Sample App with Mock service
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.core.serviceadapter;
+
+import java.util.Map;
+
+import io.vertx.core.json.JsonObject;
+
+public class ServiceAdapterResponse {
+
+  private Map<String, Object> serviceResult;
+
+  private boolean success;
+
+  private String reason;
```

Comment:
Remove this field - it does not make sense here (reason). logging is enough - reason cannot be returned to the client